server/viame_server/viame_detection.py

```diff
@@ -75,8 +75,7 @@ def export_data(self, folder):
         clipMeta = self._get_clip_meta(folder)
         detection = clipMeta.get('detection')
         if detection:
-            itemId = detection.get('_id', None)
-            export_detections = f'/api/v1/item/{itemId}/download'
```

Comment:
@AlmightyYakob your export_detections endpoint works great, I just had to change these lines to actually use it.